src/resolution/trusted-gateway-arns-resolver.ts

```diff
@@ -56,6 +56,10 @@ export class TrustedGatewayArNSResolver implements NameResolver {
       const ttl =
         parseInt(response.headers[headerNames.arnsTtlSeconds.toLowerCase()]) ||
         DEFAULT_ARNS_TTL_SECONDS;
+      const limit =
+        parseInt(response.headers[headerNames.arnsLimit.toLowerCase()]) || 10;
```

Comment:
`10` should be in a constant rather than a magic number here.

Reply:
updated in [ba0ef63](https://github.com/ar-io/ar-io-node/pull/234/commits/ba0ef6301ed876517dd14a30791918be121bda09)